aten/src/ATen/native/cuda/Math.cuh

```diff
@@ -0,0 +1,91 @@
+#pragma once
+#include <type_traits>
+
+namespace at {
+namespace native {
+
```

Comment:
I'm not sure I'd call this `Math.cuh` because it only has the digamma and trigamma functions.

Reply:
I think it make sense to leave this file, it can be used for other math operations.